en/lessons/gravity-model.md

```diff
@@ -0,0 +1,712 @@
+
+ 1. The "vagabond poor" - the stereotypical poor individual from elsewhere.
+ 2. Demobilised soldiers who used the vagrancy system for a free ride home.
+ 3. Individuals expelled back to London from other counties (not relevant to the original research question and excluded here).
+
+ The first two groups represent migrants from elsewhere, but because they came to London via quite different paths - one as an economic migrants and one dumped on the docks after finishing work overseas - they were modelled separately in the original article. Splitting these two types of vagrants into two models meant more defensible results. Those were not the ONLY two subsets we could have modelled, as any subset for which there was an intellectual case would also do (e.g. men and women).
+
+ For the sake of brevity, this tutorial will take readers through the modelling of the 3,262 "vagabond poor", though the process for modelling the demobilised soldiers is exactly the same.
+
+## Preview of the Finished Model
+
+The result of the modelling process can be seen in Figure 5. As you can see on the map, there are in fact regional anomalies. There is a cluster of counties in the West Midlands (four blue counties) that were over-sending migrants to London. There were also a number of counties in the centre of the map and towards the north that were under-sending migrants (red), and there are a few regional anomalies sprinkled around the country. The remainder of this tutorial will walk you through the process of making those types of discoveries from a set of historical data, starting with the mathematics that allow us to do this type of work.
+
+{% include figure.html filename="figure5.png" caption="Figure 5: The anomalous counties in the original study, showing areas with fewer migrants than expected, more migrants than expected, and about the expected number." %}
+
+The formula used to arrive at that result is provided below, with the following sections outlining the origins and rationale of that formula.
+
+$$μ_{ij} = exp(β_{0} + β_{1}ln(P_{i}) + β_{2}ln(d_{ij}) + β_{3}Wh_{i} + β_{4}Wa_{i} + β_{5}WaT_{i})$$
+
+# Regression Modelling, and the Mathematics Behind our Model
+
+<div class="alert alert-warning">
+
+The following section outlines the intellectual origins of the gravity model formula, and will help you to understand the branch of mathematics from which it comes. This will allow you to develop the vocabulary and background needed to discuss the model's mathematics at a basic level. If you are not familiar or comfortable with mathematical operations, you may find it helpful to read or refresh your knowledge of orders of operation, as this knowledge is needed to solve the mathematical equations correctly.
+
+<ul><li><a href="https://en.wikipedia.org/wiki/Order_of_operations">order of operations</a></li></ul>. 
```

Comment:
This looks a bit weird to be an unordered list, when it's only one link. Maybe just put it inline with the rest of the paragraph?

Reply:
changed to inline link.